src/Components/SavedPage/SavedPage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Tile from "../Tile/Tile.js"
+import './SavedPage.css'
+
+const SavedContainer = ({ wordList, removeTile, saveTile, savedWords }) => {
+
```

Comment:
can take out wordlist if you are not using it!

Reply:
Thank you! Missed that one!